workers/src/test/java/edu/cornell/testenv/testrunner/TestJUnit5TestRunner.java

```diff
@@ -0,0 +1,22 @@
+package edu.cornell.testenv.testrunner;
+
+import edu.cornell.testenv.testcontext.JUnitTestContext;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestJUnit5TestRunner {
+
+    @Test
+    public void testJUnitRunner() {
+
+        JUnitTestContext context = new JUnitTestContext( new ArrayList<>(List.of("edu.cornell.testenv.testclasses.TestClassJUnit")));
+
+        JUnit5TestRunner runner = new JUnit5TestRunner();
+        boolean result = runner.runTest(context);
```

Comment:
Should this `result` variable be checked? What happens if it's false?

Reply:
It would print "Tests passed: false", no?